source/process/bug-report.md

```diff
@@ -0,0 +1,108 @@
+# Bug Reports
+
+A bug for Mattermost is defined as "any obvious error". Errors on unsupported platforms are not considered bugs.
+
+Sample bugs:
+   - The front page is blank.
+   - /msg does not work for users outside of the current team.
+   - Whitespace is removed beside emoticons.
+
+Sample non-bugs:
+   - Sort channels based on custom ordering.
+   - Mattermost does not have a native app for Windows Phone.
+   - @all does not work.
+
+## Bug Report Format
+
+A well-written bug report consists of several components:
+   1. **Summary**, the bug in one sentence.
+   2. **Reproduction Steps**, how to reproduce the bug.
+   3. **Expected Behavior**, describe the bug in detail.
+   4. **Observed Behavior**, what exactly happened.
+   5. **Possible Fixes** (optional), linking code/settings that may have caused the bug.
+
+## How to File A Good Bug Report
+
+The most important part of a bug report is the text, if a developer cannot tell what you meant, then the bug report was useless. Well written bug reports are:
+   - **Absolute**, if something can be done in two different ways, state exactly which one you used.
+   - **Verbose**, giving more information is almost always more useful than less.
+   - **Explicit**, avoid the use of pronouns. Words such as "it", or references such as "the box" are often unclear.
+
+### The Summary
+
+Be as clear as you can in the summary, a good summary tells the reader:
+   - **What** happened? Be as exact as you can (e.g. The app crashes)
+   - **How** did it happen? Give a brief description of the process used (e.g. when switching channels from Town Square to any other channel)
+   - **Where** does it happen? Include the version number/platform. (e.g. on Android v5.1 running Mattermost v3.0.2.)
+   
+### Reproduction Steps
+
+State exactly which actions you took before encountering the bug. A good list of reproduction steps is:
+   - **Clear**, anyone should be able to reproduce the bug using just your instructions
+   - **Accurate**, no vague descriptions should be used. (e.g. "Switch channel to Off-Topic" vs. "Switch channel to Off-Topic using ALT+UP")
```

Comment:
"Switch channel to Off-Topic using ALT+UP" maybe switch to "Switch channel to Off-Topic by clicking on the channel in the LHS" 0/5